data/ground_truth.py

```diff
@@ -2,8 +2,251 @@
 # TODO(Georg): add license
 import rospy
 
+class Transform:
+    def __init__(self, x = 0 ,y = 0 ,z = 0):
+        self.x = x
+        self.y = y
+        self.z = z
+
+    def __mul__(self, other):
+        self.x += other.x
+        self.y += other.y
+        self.z += other.z
+        return self
+
+    def __str__(self):
+        return ("x: " + str(self.x) + " y: " + str(self.y) + " z: " + str(self.z))
+
+class Frame:
+    def __init__(self, name, transform):
+        self.name = name
+        self.transform = transform
+        self.children = []
+        self.allChildren = []
+
+
+class GroundTruth:
+
+    def __init__(self):
+        self.root = Frame("root", Transform(0,0,0))
+        #self.root.allChildren.append("root")
+
+    def addFrame(self, parent_frame_name, child_frame_name, transform):
```

Comment:
Please rename into ```add_node```. 

Also, please note the Python style guide for naming conventions: "Function names should be lowercase, with words separated by underscores as necessary to improve readability." (https://www.python.org/dev/peps/pep-0008)